src/nvim/globals.h

```diff
@@ -1013,6 +1013,10 @@ EXTERN char e_line_number_out_of_range[] INIT(= N_("E1247: Line number out of ra
 
 EXTERN char e_highlight_group_name_too_long[] INIT(= N_("E1249: Highlight group name too long"));
 
+// clint whines about line length if all of it is in the same line
```

Comment:
good to know, but the extra comment makes it worse :) It's mostly implied (and thus redundant) that the codebase is guarded by a linter, so don't need to mention it.

Reply:
Fixed, sorry for the comment, thought I needed to clarify since the line break looked a bit weird to me